pkg/gameservers/portallocator_test.go

```diff
@@ -39,6 +39,37 @@ var (
 	n3 = corev1.Node{ObjectMeta: metav1.ObjectMeta{Name: "node3", UID: "node3"}}
 )
 
+func TestEmptyPortPolicy(t *testing.T) {
```

Comment:
I think this test name is not accurate. It should describe which function do you test.
There is a test function TestPortAllocatorAllocate, so you can put this scenario there as a separate `t.Run("empty port policy")` statement.